runsc/boot/loader.go

```diff
@@ -479,6 +479,9 @@ func createMemoryFile() (*pgalloc.MemoryFile, error) {
 }
 
 func (l *Loader) installSeccompFilters() error {
+	if l.conf.CoverageFilesDirectory != "" {
+		l.conf.DisableSeccomp = true
+	}
```

Comment:
Why is this necessary? If we absolutely need it, please add a comment to the flag mentioning that coverage will turn off seccomp.